src/webpack.config.js

```diff
@@ -32,11 +33,24 @@ module.exports = {
 		],
 		loaders: [
 			{ test: /src[\\\/].*\.ts?$/, loader: 'umd-compat-loader!ts-loader' },
-			{ test: /\.js?$/, loader: 'umd-compat-loader' },
+			{ test: /\.js?$/, loader: 'umd-compat-loader'  },
 			{ test: /\.html$/, loader: 'html' },
 			{ test: /\.(jpe|jpg|png|woff|woff2|eot|ttf|svg)(\?.*$|$)/, loader: 'file?name=[path][name].[hash:6].[ext]' },
-			{ test: /\.styl$/, loader: ExtractTextPlugin.extract(['css-loader?sourceMap', 'stylus-loader']) },
-			{ test: /\.css$/, loader: 'style-loader!css-loader?modules' },
+			{
+				test: /\.styl$/,
+				exclude: /\.module\.styl$/,
+				loader: ExtractTextPlugin.extract([
```

Comment:
hoist the `ExtractTextPlugin` call to a `const` and then we can put all this back onto one line. Please do this for both of the uses. ie.

``` javascript
const stylusLoader = ExtractTextPlugin.extract([
	'css-loader?sourceMap',
	'stylus-loader?sourceMap'
]);


loaders: [
	// ...
	{ test: /\.styl$/, exclude: /\.module\.styl$/, loader: stylusLoader }
	// ...
]
```